src/sdkV2.ts

```diff
@@ -104,22 +127,133 @@ export default class CegaEvmSDKV2 {
     });
   }
 
-  async bulkSettleVaultsDcs(
+  async bulkProcessDepositQueuesDcs(
     vaultAddresses: EvmAddress[],
+    maxProcessCount: ethers.BigNumber,
+    overrides: TxOverrides = {},
   ): Promise<ethers.providers.TransactionResponse> {
     const cegaEntry = this.loadCegaEntry();
-    return cegaEntry.bulkSettleDCSVaults(vaultAddresses);
+    return cegaEntry.bulkProcessDCSDepositQueues(vaultAddresses, maxProcessCount, {
+      ...(await this._gasStation.getGasOraclePrices()),
+      ...overrides,
+    });
   }
 
-  async bulkStartTradesDcs(
+  async endAuctionDcs(
+    vaultAddress: EvmAddress,
+    auctionWinner: EvmAddress,
+    tradeStartDate: ethers.BigNumber,
+    aprBps: ethers.BigNumber,
+    oracleDataSource: OracleDataSourceDcs,
+    overrides: TxOverrides = {},
+  ): Promise<ethers.providers.TransactionResponse> {
+    if (!this._signer) {
+      throw new Error('Signer not defined');
+    }
+
+    const cegaEntry = this.loadCegaEntry();
+    return cegaEntry.endDCSAuction(
+      vaultAddress,
+      auctionWinner,
+      tradeStartDate,
+      aprBps,
+      oracleDataSource,
+      {
+        ...overrides,
+      },
+    );
+  }
+
+  /**
+   * SETTLEMENT METHODS
+   */
+
+  async bulkCollectFeesDcs(
     vaultAddresses: EvmAddress[],
+    overrides: TxOverrides = {},
   ): Promise<ethers.providers.TransactionResponse> {
     const cegaEntry = this.loadCegaEntry();
-    return cegaEntry.bulkStartDCSTrades(vaultAddresses);
+    return cegaEntry.bulkCollectDCSFees(vaultAddresses, {
+      ...(await this._gasStation.getGasOraclePrices()),
+      ...overrides,
+    });
   }
 
-  async submitDispute(vaultAddress: EvmAddress): Promise<ethers.providers.TransactionResponse> {
+  async bulkProcessWithdrawalQueuesDcs(
+    vaultAddresses: EvmAddress[],
+    maxProcessCount: ethers.BigNumber,
+    overrides: TxOverrides = {},
+  ): Promise<ethers.providers.TransactionResponse> {
     const cegaEntry = this.loadCegaEntry();
-    return cegaEntry.submitDispute(vaultAddress);
+    return cegaEntry.bulkProcessDCSWithdrawalQueues(vaultAddresses, maxProcessCount, {
+      ...(await this._gasStation.getGasOraclePrices()),
+      ...overrides,
+    });
+  }
+
+  async bulkRolloverVaultsDcs(
+    vaultAddresses: EvmAddress[],
+    overrides: TxOverrides = {},
+  ): Promise<ethers.providers.TransactionResponse> {
+    const cegaEntry = this.loadCegaEntry();
+    return cegaEntry.bulkRolloverDCSVaults(vaultAddresses, {
+      ...(await this._gasStation.getGasOraclePrices()),
+      ...overrides,
+    });
+  }
+
+  /**
+   * DISPUTE METHODS
+   */
+
```

Comment:
there was a dispute method above also. move that down here

Reply:
moved